index.html

```diff
@@ -211,6 +214,68 @@ <h3 class="h-title">About Myself</h3>
 </ul>
 <a href="#" class="last-button">Get my resume</a>
 </div>
+<div class="languages">
+  <ul>
+    <li>
+      <div class="arrow">
+        <p id="lang">languages</p>
+        <div class="arrow-pointing-down"></div>
+      </div>
+    <ul class="dropdown-items">
+      <li><img src="img/Ellipse1.png" ><a href="#"><span>Java Script</span></a></li>
+      <li><img src="img/Ellipse2.png"><a href="#">HTML</a></li>
+      <li><img src="img/Ellipse3.png"><a href="#">CSS</a></li>
+    </ul>
+  
+
+  </li>
+  </ul>
+  <ul>
+    <li>
+      <div class="arrow">
+          <p id="lang">Frameworks</p>
+        <div class="arrow-right"></div>
+      </div>
+    </li>
+  </ul>
+    <hr>
+    <ul>
+      <li>
+        <div class="arrow">
+            <p id="lang">Skills</p>
+          <div class="arrow-right"></div>
+        </div>
+      </li>
+    </ul> 
+    <hr>
+     
+</div>
+</div>
+</section>
+<section class="contact-section">
+  <div class="contacts">
+    <h3 class="heading-two">contact me</h3>
+    <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+      I’d love to help with it!</p>
+    <form  action="https://formspree.io/f/xknaegpd" method="post" class="contact-form">
+      <ul>
+      <li>
+      <input type="text" maxlength="30" id="name" name="name" placeholder="enter your username" required>
+      </li>
+      <li>
+        <input type="email" name="email" placeholder="enter your email " required>
+      </li>
+      <li>
+        <textarea id="comment" maxlength="500" name="comment" placeholder="write your comment" required></textarea>
+      </li>
+      <li>
+        <button  class="button project"type="submit">Get in touch</button>
+        </li></ul>
+    
```

Comment:
- [ ] [REQUIRED] Based on the project requirement, you should use Flexbox to align the contents of this contact section. Kindly use flexbox to layout this section.